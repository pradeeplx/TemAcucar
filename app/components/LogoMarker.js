import React from 'react'
import { Image, PixelRatio } from 'react-native'

export default LogoMarker = ({ style }) => {
  const ratio = PixelRatio.get()
  let image
  if (ratio <= 1) {
    image = require('../img/marker100.png')
  } else if (ratio <= 1.5) {
    image = require('../img/marker150.png')
  } else if (ratio <= 2) {
    image = require('../img/marker200.png')
  } else if (ratio <= 3) {
    image = require('../img/marker300.png')
  } else if (ratio <= 3.5) {
    image = require('../img/marker350.png')
  } else {
    image = require('../img/marker400.png')
  }
  return (
    <Image source={image} style={[{
      width: 15,
      height: 15,
    }, style]}/>
  )
}
