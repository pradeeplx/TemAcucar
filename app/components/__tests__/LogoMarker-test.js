import 'react-native'
import React from 'react'
import LogoMarker from '../LogoMarker'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <LogoMarker style={{top: 0}}>FooBar</LogoMarker>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
