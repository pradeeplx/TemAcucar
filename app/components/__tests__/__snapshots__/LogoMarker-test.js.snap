exports[`test renders correctly 1`] = `
<Image
  source={1}
  style={
    Array [
      Object {
        "height": 15,
        "width": 15,
      },
      Object {
        "top": 0,
      },
    ]
  } />
`;
